OlinkAnalyze/inst/WORDLIST

```diff
@@ -192,4 +207,4 @@ PlateLQL
 Quant
 Topouza's
```

Comment:
Is this something we should fix? I don't see anyone else's name, not sure where this came from.

Reply:
Its because your name is in the  News (since we added you to the author list)